### PR TITLE
[ADD] account:  rainbow man when finishing to configure the accountin…

### PR DESCRIPTION
**Pull request by ppr-odoo**
…g via the new config bar

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
changed the target to master

**Comment by rgo-odoo** [comment]
LGTM

**Comment by qdp-odoo** [comment]
naaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaann :sob:

File: addons/account/wizard/setup_wizards.py [hunk 1]
```diff
@@ -60,6 +60,14 @@ class OpeningAccountMoveWizard(models.TransientModel):
 
     def validate(self):
         self.opening_move_id.post()
+        return {
+            "effect": {
+                'fadeout': 'slow',
+                'img_url': '/web/static/src/img/smile.svg',
+                'message': "Well done! The configuration steps have been done successfully.",
```
Comment (qdp-odoo) [inline addons/account/wizard/setup_wizards.py]: btw, there's a missing _

**Comment by tivisse** [comment]
You're reluctant to change.

File: addons/account/wizard/setup_wizards.py (hunk 1 above)
Comment (tivisse) [reply addons/account/wizard/setup_wizards.py]: Ok sir.

**Comment by qdp-odoo** [comment]
to this one? exactly! :zipper_mouth_face: ^^

**Pull request merged**